### DISCUSSION TITLE
Availability

### DISCUSSION THREAD

**Discussion by SummerSky777** [decheine/complete-compendium#224] [General]
This is great and i have been using the website since it begun. Could you make the site available for download in a simple manner, single link in a prominent place with simple file and pretty much easy to use for the average person that is not familiar with programming and websites and the such, for offline use, updated every few months or a year? The lomion de website was lost as it did not have this feature. 

Thank you.

**Comment by decheine** [comment]
Hi there, thank you for this recommendation. This is something that's been on my list of things to do for this but I haven't gotten around to it. The plan is to create a "Release" that is simply a zip file that contains all of the mosnter html pages and images that is browsable on it's own, although it might not look as good as the live website. There have been some issues with this that make it not super straight forward so it'll have to build something that generates the release if I want to to work well. 

I'll put this at the top of the priority list, since it would certainly benefit a lot of people.

**Comment by SummerSky777** [comment]
Thanks again. The lomion de website was very useful, and fun to browse, and this is a colossal upgrade from it. We should be able to preserve it and just like you write in a simple file (as some of us kind of suck at these computer stuff). Even if it does not look as pretty on the offline version it would still have all the useful material.

**Reply by decheine** [reply]
I've pushed a release, you should see it on the main page of this repository labeled "Offline Compendium"

**Comment by SummerSky777** [comment]
Wonderful!
